docs/user/dql/functions.rst

```diff
@@ -1425,6 +1425,29 @@ Example::
     | 26                               |
     +----------------------------------+
 
+DAY_OF_MONTH
+----------
```

Comment:
```suggestion

DAY_OF_MONTH
------------
```

Reply:
Addressed in 469fbb23ed6003676fa19f98c4c80e838ba3d949